tests/lib/tools/systemd-state

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash
+
+show_help() {
+    echo "usage: create-and-start-unit <unit-name> <unit-args> [--overrides <conf>] [--persistent]"
+    echo "       stop-and-remove-unit <unit-name>"
+    echo "       wait-for-unit <unit-name>"
+    echo "       stop-units <unit-names>"
+    echo "       active-snapd-units"
+    echo ""
+    echo "Manage systemd service units and retrieve information about existing units."
+    echo ""
+    echo "COMMANDS:"
+    echo "  create-and-start-unit:   creates and starts systemd unit. In case it is set as persistent the unit survive reboots. Also it allows to overrides the configuration."
```

Comment:
```suggestion
    echo "  create-and-start-unit:   creates and starts systemd unit. In case it is set as persistent the unit survive reboots. Also it allows to override the configuration."
```